README.md

```diff
@@ -26,12 +26,13 @@ jobs:
     - uses: pkgjs/meet@v0
       with:
         token: ${{ secrets.GITHUB_TOKEN }}
-        schedules: 2020-04-02T17:00:00.0Z/P1D
+        schedules: 2020-04-02T17:00:00.0Z/P1D # Every 1 day from 2020-04-02 at 5PM UTC
+        createWithin: P2D # Create issue 2 days before the scheduled meeting
```

Comment:
I added createWithin to the main example because it is marked as required and is generally something folks will want to set